addons/editor/stringtable.xml

```diff
@@ -159,6 +159,9 @@
             <Polish>Wysadza pasażerów ze wszystkich wybranych pojazdów. Jeżeli pojazd jest w powietrzu, pasażerowie dostaną spadochron.\nJednostki zostaną wysadzone jedna po drugiej (nie wszystkie na raz).</Polish>
             <Japanese>選択された全車両から後部座席のユニットを脱出させます。もし車両が飛行中の場合はパラシュートが与えられます。\nなお脱出は一人づつ降りるようになっています (即時ではなく数秒づつ)。</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Editor_UnloadViV_Description">
+            <English>Unloads all selected Vehicle-in-Vehicle cargo and carriers. A vehicle that is simultaneously both is unloaded from its carrier.</English>
```

Comment:
```suggestion
            <English>Unloads all selected vehicle-in-vehicle cargo and carriers. A vehicle that is simultaneously both is unloaded from its carrier.</English>
```
Vehicle-in-vehicle is nothing special, I see no need to capitalize it.

Reply:
86e368d